classes/PodsI18n.php

```diff
@@ -313,15 +313,26 @@ public function get_current_language_data( $args = array() ) {
 				$current_language = get_user_meta( get_current_user_id(), 'pll_filter_content', true );
 			}
 
+			$pods_ajax = pods_v( 'pods_ajax', 'request', false );
+
 			// Get current language based on the object language if available.
 			$page = basename( $_SERVER['SCRIPT_NAME'] );
 
+			if ( $pods_ajax && 'admin-ajax.php' === $page ) {
+				$page = basename( $_SERVER['HTTP_REFERER'] );
```

Comment:
* :warning: Detected usage of a non-sanitized input variable: $_SERVER['HTTP_REFERER']
* :no_entry_sign: Detected usage of a possibly undefined superglobal array index: $_SERVER['HTTP_REFERER']. Use isset() or empty() to check the index exists before using it

<sub>hash: cd3e0b60c619c6310a3c7214ddf2147c</sub>